src/pages/HomePage/pages/MarketPlace/components/forms/ProductForm.tsx

```diff
@@ -207,7 +99,7 @@ export function ProductForm({
                   placeholder="Select status"
                 />
               </div>
-            </div>
+            </FormLayout>
 
             {/* Submit Button */}
             <div className="flex justify-end gap-3 border-t p-2">
```

Comment:
for this buttons please use Actions compononent
import { Actions } from "@/components/ui/form/Actions";